matlab/crf_correction.m

```diff
@@ -13,20 +13,30 @@
 %           [It,x] = crf_correction(I, Igt, 3, 0.01, 'pq', 'luv');
 
 function [It,x] = crf_correction(Ir, Igt, varargin)
-    args = {3,0.01,'pq','luv'};  % default values
+    args = {3,0.01,'pq','luv',0};  % default values
```

Comment:
Gabriel, please add 'normalization' to the function documentation above. 

You can also consider using the new fancy Matlab syntax for defining arguments and their default values:
https://uk.mathworks.com/help/matlab/ref/arguments.html

Reply:
Sure, added some info. I can update with new Matlab arguments later if this would be more convenient.